src/ducks/loans/contractInfo.js

```diff
@@ -0,0 +1,87 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import snxJSConnector from '../../utils/snxJSConnector';
+import { bigNumberFormatter } from '../../utils/formatters';
+
+import { CRYPTO_CURRENCY_MAP, SYNTHS_MAP } from '../../constants/currency';
+
+export const contractInfoSlice = createSlice({
+	name: 'contractInfo',
+	initialState: {
+		collateralPair: null,
+		loadingError: null,
+		isLoading: false,
+		isLoaded: false,
+		isRefreshing: false,
+	},
```

Comment:
note: in future PRs I'll probably abstract all this request stuff into a middleware... so we won't have to repeat this code all over... for now, its the quickest solution.